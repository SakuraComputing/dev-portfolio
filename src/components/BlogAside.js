import React from 'react';

export const BlogAside = () => (
    <aside className="blog aside section">
        <div className="section-inner">
            <h2 className="heading">Latest Blog Posts</h2>
            <p>You can use Sascha Depold's <a href="https://github.com/sdepold/jquery-rss" target="_blank">jQuery RSS
                plugin</a> to pull in your blog post feeds.</p>
            <div id="rss-feeds" className="content">

            </div>
        </div>
    </aside>
);
export default BlogAside;