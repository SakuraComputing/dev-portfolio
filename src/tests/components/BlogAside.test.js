import React from 'react';
import { shallow } from 'enzyme';
import BlogAside from '../../components/BlogAside';

describe('BlogAside Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<BlogAside />);
        expect(wrapper).toMatchSnapshot();
    });
});