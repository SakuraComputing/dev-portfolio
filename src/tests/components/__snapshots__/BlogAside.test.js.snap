// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogAside Section should render correctly 1`] = `
<aside
  className="blog aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Latest Blog Posts
    </h2>
    <p>
      You can use Sascha Depold's 
      <a
        href="https://github.com/sdepold/jquery-rss"
        target="_blank"
      >
        jQuery RSS plugin
      </a>
       to pull in your blog post feeds.
    </p>
    <div
      className="content"
      id="rss-feeds"
    />
  </div>
</aside>
`;
